packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -169,7 +169,7 @@ export class FrontendPluginImpl {
     const functionPlugin = ctx.configOfOtherPlugins.get(DependentPluginInfo.FunctionPluginName);
     if (functionPlugin) {
       functionEnv = {
-        defaultName: functionPlugin.get(DependentPluginInfo.FunctionDefaultName) as string,
+        defaultName: ctx.projectSettings?.defaultFunctionName as string,
```

Comment:
I think in latest dev branch, `defaultFunctionName` is also used in frontend's pre-deploy method. Please rebase and also change that one. Thanks!

Reply:
Thanks for reminder. 
I've pulled the latest dev branch and updated the frontend plugin pre-deploy in the latest commit.